scripts/install

```diff
@@ -29,146 +24,142 @@ domain=$YNH_APP_ARG_DOMAIN
 
 # Create a system user
-ynh_system_user_create $app
+ynh_system_user_create --username=$app
+
+#=================================================
+# PHP-FPM CONFIGURATION
+#=================================================
+ynh_script_progression --message="Configuring PHP-FPM..." --weight=3
+
+# Create a dedicated PHP-FPM config
+ynh_add_fpm_config --package="$extra_php_dependencies"
+phpversion=$(ynh_app_setting_get --app="$app" --key=phpversion)
 
 #=================================================
 # MODIFY A CONFIG FILE
 #=================================================
 
-cp -a ../conf/config_inc.php.sample $final_path/config/config_inc.php.sample
+cp -a ../conf/config_inc.php $final_path/config/config_inc.php
 
-ynh_replace_string "yunouser"   "$db_name"    "$final_path/config/config_inc.php.sample"
-ynh_replace_string "yunopass"   "$db_pwd"     "$final_path/config/config_inc.php.sample"
-ynh_replace_string "yunobase"   "$db_name"    "$final_path/config/config_inc.php.sample"
-#ynh_replace_string "yunomail"   "$email"      "$final_path/config_inc.php.sample"
-#ynh_replace_string "yunodomain" "$domain"     "$final_path/config_inc.php.sample"
+ynh_replace_string --match_string="__RANDOM__" --replace_string="$(ynh_string_random --length=20)" --target_file="$final_path/config/config_inc.php"
+ynh_replace_string --match_string="__DB_NAME__" --replace_string="$db_name" --target_file="$final_path/config/config_inc.php"
+ynh_replace_string --match_string="__DB_USER__" --replace_string="$db_user" --target_file="$final_path/config/config_inc.php"
+ynh_replace_string --match_string="__DB_PWD__" --replace_string="$db_pwd"  --target_file="$final_path/config/config_inc.php"
+ynh_replace_string --match_string="__DOMAIN__" --replace_string="$domain"  --target_file="$final_path/config/config_inc.php"
 
 #=================================================
 # STORE THE CONFIG FILE CHECKSUM
 #=================================================
 
-### `ynh_store_file_checksum` is used to store the checksum of a file.
-### That way, during the upgrade script, by using `ynh_backup_if_checksum_is_different`,
-### you can make a backup of this file before modifying it again if the admin had modified it.
-
-# Calculate and store the config file checksum into the app settings
-#ynh_store_file_checksum "/etc/onlyoffice/documentserver/nginx/onlyoffice-documentserver.conf"
-
+ynh_store_file_checksum --file="$final_path/config/config_inc.php"
 
 #=================================================
 # GENERIC FINALIZATION
 #=================================================
 # SECURE FILES AND DIRECTORIES
 #=================================================
 
-### For security reason, any app should set the permissions to root: before anything else.
-### Then, if write authorization is needed, any access should be given only to directories
-### that really need such authorization.
-
 # Set permissions to app files
 chown -R www-data:www-data $final_path
```

Comment:
Not tested, but, why the doesn't `final_path` belong to the app user?
```suggestion
chown -R $app: $final_path
```
(If you apply this change, please change it in other scripts)